!!! compile failed ,media-driver-intel-media-21.1.3【centos7,gcc (GCC) 8.3.1 20190311 (Red Hat 8.3.1-3)】!!!,

HI：
i'm getting the following build error ,[media-driver-intel-media-21.1.3]
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
[root@localhost build]# make
[  2%] Built target igfxcmrt
[  2%] Built target iHD_drv_video_SSE4
[  6%] Built target igfx_gmmumd_dll
[  6%] Building CXX object media_driver/CMakeFiles/iHD_drv_video_OBJ.dir/agnostic/common/cm/cm_array.cpp.o
In file included from /home/oscar/IntelMedia2113/media-driver-intel-media-21.1.3/media_driver/linux/common/cm/hal/cm_mem_os.h:30:0,
                 from /home/oscar/IntelMedia2113/media-driver-intel-media-21.1.3/media_driver/agnostic/common/cm/cm_mem.h:122,
                 from /home/oscar/IntelMedia2113/media-driver-intel-media-21.1.3/media_driver/agnostic/common/cm/cm_array.cpp:28:
/usr/lib/gcc/x86_64-redhat-linux/4.8.5/include/smmintrin.h:31:3: error: #error "SSE4.1 instruction set not enabled"
 # error "SSE4.1 instruction set not enabled"
   ^
cc1plus: error: unrecognized command line option "-Wno-delete-incomplete" [-Werror]
cc1plus: all warnings being treated as errors
make[2]: *** [media_driver/CMakeFiles/iHD_drv_video_OBJ.dir/build.make:63: media_driver/CMakeFiles/iHD_drv_video_OBJ.dir/agnostic/common/cm/cm_array.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:249: media_driver/CMakeFiles/iHD_drv_video_OBJ.dir/all] Error 2
make: *** [Makefile:150: all] Error 2

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

--- a/cmrtlib/linux/CMakeLists.txt
+++ b/cmrtlib/linux/CMakeLists.txt
@@ -33,15 +33,9 @@ endif()
 
 # Set up compile options that will be used for the Linux build
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CPP_STANDARD_OPTION} -fPIC -fpermissive -fstack-protector-all -Werror")
-if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.1")
-endif()
 set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fno-strict-aliasing -D_FORTIFY_SOURCE=2")
 set(CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS_DEBUG} -D_DEBUG -D__DEBUG -O0")
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${CPP_STANDARD_OPTION} -fPIC -fpermissive -fstack-protector-all -Werror")
-if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -msse4.1")
-endif()
 set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -fno-strict-aliasing -D_FORTIFY_SOURCE=2")
 set(CMAKE_C_FLAGS_DEBUG   "${CMAKE_C_FLAGS_DEBUG} -D_DEBUG -D__DEBUG -O0")
 

